tests/drop_counters/mock_server.py

```diff
@@ -0,0 +1,107 @@
+"""
+Utilities for mocking a server using PTF.
+"""
+
+import logging
+import time
+
+from common.platform.device_utils import fanout_switch_port_lookup
+
+class MockServer(object):
+    """
+    Mocks the presence of a server connected to the DUT.
+
+    "Starting" the server will populate the FDB and ARP tables, allowing for realistic traffic
+    forwarding behavior in the DUT.
+    """
+    def __init__(self, server_addr, server_prefix, neighbor_addr, neighbor_iface, neighbor_port,
+                 ptfhost, fanouthosts):
+        """
+        Initializes the server.
+
+        Args:
+            server_addr (IPAddress): The IP address of the server.
+            server_prefix (int): The network prefix of the server.
+            neighbor_addr (IPAddress): The IP address of the device the server is connected to
+                (typically an IP interface in SONiC).
+            neighbor_iface (str): The interface associated with the `neighbor_addr`.
+            neighbor_port (int): The port number associated with the `neighbor_iface`.
```

Comment:
This is not really neighbor_port, this is port index on ptf. The naming of this variable is a bit confusing.